apps/modernization-ui/src/pages/patient/search/result/PatientResult.tsx

```diff
@@ -37,7 +37,7 @@ const PatientResult = ({ result, onSelected }: PatientResultProps) => {
                             }}>
                             {!result.legalName
                                 ? 'No Data'
-                                : formattedName(result.legalName?.last, result.legalName?.first)}
+                                : displayName('full', result?.legalName)}
```

Comment:
The `displayName` function returns a function to use for displaying names.  So, ` displayName()` or `displayName('full')` will return a function that given a `DisplayableName` then it will format it as a full name.

There are two ways you can call it here.  `displayName()(result.legalName)` or

```
const patientName = displayName();

....


patientName(result.legalName);
```

Reply:
Thank you for the code review.